test/config/integration/server_xds.bootstrap.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
re: (2) I think we need to think about this more. I don't see any technical reason that it can't be a single file with a bit of change, and for "hello world" and even some complex deployments it is vastly simpler to manage. Anyway I think this is OK for now but we should track and think about.